oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/EBPFReceiverProvider.java

```diff
@@ -65,12 +67,19 @@ public void prepare() throws ServiceNotProvidedException, ModuleStartException {
 
     @Override
     public void start() throws ServiceNotProvidedException, ModuleStartException {
+        // load official analysis
+        getManager().find(CoreModule.NAME)
+            .provider()
+            .getService(OALEngineLoaderService.class)
+            .load(EBPFOALDefine.INSTANCE);
+
         final GRPCHandlerRegister grpcHandlerRegister = getManager().find(SharingServerModule.NAME)
                 .provider()
                 .getService(GRPCHandlerRegister.class);
         grpcHandlerRegister.addHandler(new EBPFProcessServiceHandler(getManager()));
         grpcHandlerRegister.addHandler(new EBPFProfilingServiceHandler(getManager()));
         grpcHandlerRegister.addHandler(new ContinuousProfilingServiceHandler(getManager(), this.config));
+        grpcHandlerRegister.addHandler(new AccessLogServiceHandler(getManager()));
```

Comment:
Please refer to https://github.com/apache/skywalking/pull/11749, what we learned from that performance benchmark is that, for heavy traffic service, especially like ALS, we should have an option to provide a separate port to help traffic limitation and control.

Reply:
Updated.